src/scss/components/_cards.scss

```diff
@@ -11,16 +11,16 @@
     * https://fozzie.just-eat.com/styleguide/ui-components/cards
     */
 
-    $card-bgColor--active                           : $white;
-    $card-bgColor--disabled                         : $grey--lighter;
+    $card-bgColor--active                           : $color-white;
+    $card-bgColor--disabled                         : $color-grey-20;
     $card-tooltip-width                             : 10px;
     $card-arrow-bottom-position                     : 0;
     $card-padding                                   : spacing(x2);
     $card-radius                                    : 4px;
     $card-borderColor                               : $color-border;
     $card-section-margin                            : spacing(x4);
     $card-section-margin--large                     : spacing(x8);
-    $card-section-highlight-backgroundColor         : $orange--offWhite;
+    $card-section-highlight-backgroundColor         : $color-orange-10;
     $card-section-highlight-color                   : $color-text;
```

Comment:
I was just looking at upgrading to the latest beta version on menu and I see errors around the `$color-text` variable, it looks like it no longer exists in PIE, is that correct? And if so what should it be replace with?

Reply:
@DamianMullins I checked that it should be `$color-content-default` now.

Reply:
Thanks @dandel10n, I'll update this as it was causing issues for me.

Reply:
OK, it looks like there are other variables that also to be updated, I'll dig around for any tickets around this work. It feels like something that Ash probably already had in progress.